app/views/report/_form_styling.html.haml

```diff
@@ -36,7 +36,7 @@
               - break if s_idx > 0 && (styles[s_idx - 1].blank? || styles[s_idx - 1][:operator] == "DEFAULT")
               - if s_idx > 0
                 %br
-              - opts = [[styles[s_idx] ? "<#{_('Remove')}>" : "<#{_('Normal')}>", nil]] + ReportHelper::STYLE_CLASSES.invert.to_a.sort_by { |a| a.first.downcase }
+              - opts = [[styles[s_idx] ? "<#{_('Remove')}>" : "<#{_('Normal')}>", nil]] + Array(ReportHelper::STYLE_CLASSES.invert).map{|x| [_(x[0]), x[1]]}.to_a.sort_by { |a| a.first.downcase }
```

Comment:
The `Array` bit is not necessary - .map should work directly off of the .invert

Reply:
Additionally, since the .map is introduced, that will automatically convert it to an Array, so the .to_a is no longer necessary.